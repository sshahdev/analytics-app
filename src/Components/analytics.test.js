import React from 'react';
import { shallow, mount } from 'enzyme';
import Analytics from './analytics';

import dashboardStore from '../store/index.js';


describe('Test App', () => {
  let component, appWrapper;
  beforeEach(()=>{
      component = mount(<Analytics />);
      appWrapper = component.find('Analytics');
  });

  it('should test Analytics component', () => {
    expect(appWrapper).toMatchSnapshot();
  });

  it('length', () => {
      expect(appWrapper).toHaveLength(1);
  }); 
})