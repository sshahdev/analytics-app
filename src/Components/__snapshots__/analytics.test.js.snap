// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App should test Analytics component 1`] = `
<Analytics>
  <div
    className="App"
  >
    <Navbar
      bg="dark"
      collapseOnSelect={false}
      expand={true}
      variant="dark"
    >
      <nav
        className="navbar navbar-expand navbar-dark bg-dark"
      >
        <NavbarBrand
          href="#"
        >
          <a
            className="navbar-brand"
            href="#"
          >
            Analytics Dashboard
          </a>
        </NavbarBrand>
      </nav>
    </Navbar>
    <Container
      fluid={false}
    >
      <div
        className="container"
      >
        <Row
          noGutters={false}
        >
          <div
            className="row"
          >
            <Col
              sm={6}
            >
              <div
                className="col-sm-6"
              >
                <Card
                  body={false}
                  className="mt-5 justify-content-center"
                >
                  <div
                    className="mt-5 justify-content-center card"
                  >
                    <CardBody>
                      <div
                        className="card-body"
                      >
                        <Container
                          className="text-align-center"
                          fluid={false}
                        >
                          <div
                            className="text-align-center container"
                          >
                            <Row
                              noGutters={false}
                            >
                              <div
                                className="row"
                              >
                                <Col
                                  sm="4"
                                >
                                  <div
                                    className="col-sm-4"
                                  >
                                    Temperature
                                  </div>
                                </Col>
                                <Col
                                  sm="8"
                                >
                                  <div
                                    className="col-sm-8"
                                  >
                                    0
                                  </div>
                                </Col>
                              </div>
                            </Row>
                            <Row
                              noGutters={false}
                            >
                              <div
                                className="row"
                              >
                                <Col
                                  sm="4"
                                >
                                  <div
                                    className="col-sm-4"
                                  >
                                    Air pressure
                                  </div>
                                </Col>
                                <Col
                                  sm="8"
                                >
                                  <div
                                    className="col-sm-8"
                                  >
                                    0
                                  </div>
                                </Col>
                              </div>
                            </Row>
                            <Row
                              noGutters={false}
                            >
                              <div
                                className="row"
                              >
                                <Col
                                  sm="4"
                                >
                                  <div
                                    className="col-sm-4"
                                  >
                                    Humidity
                                  </div>
                                </Col>
                                <Col
                                  sm="8"
                                >
                                  <div
                                    className="col-sm-8"
                                  >
                                    0
                                  </div>
                                </Col>
                              </div>
                            </Row>
                          </div>
                        </Container>
                      </div>
                    </CardBody>
                  </div>
                </Card>
              </div>
            </Col>
          </div>
        </Row>
      </div>
    </Container>
  </div>
</Analytics>
`;
